CONTRIBUTING.md

```diff
@@ -305,10 +316,9 @@ Within Podman Desktop, we use the following frameworks and tools to build the de
 ### Testing
 
 Within Podman Desktop, we use the following for testing:
-
```

Comment:
I suppose this blank line was added by prettier

Reply:
Seems like it.